llvm/docs/ReleaseNotes.rst

```diff
@@ -95,15 +95,45 @@ Changes to LLVM infrastructure
   `RFC  <https://discourse.llvm.org/t/rfc-ffat-lto-objects-support/63977>`_
   for more details.
 
+* ``ValueTypes.td`` is made richer to represent attributes for
+  ``MVT``.
+
+* ``Intrinsics.td`` is able to emit definitions of
+  ``TypeSig``. ``IntrinsicEmitter`` doesn't depend on
+  ``MachineValueTypes.h`` anymore.
+
+* ``llvm/CodeGen/MachineValueType.h`` is moved from ``llvm/Support``.
+  It uses values defined in ``ValueTypes.td``.
+
 Changes to building LLVM
 ------------------------
 
+* ``llvm-min-tblgen`` is internally introduced to build LLVM public
+  headers. Note that ``llvm-tblgen`` depends on `GenVT.inc` that is
+  generated by ``llvm-min-tblgen``. Specify the external
```

Comment:
How about

> Builds using ``LLVM_TABLEGEN`` to specify a pre-built tablegen executable should use ``llvm-tblgen``. 

I think you can drop the part about ``llvm-tblgen`` being a superset of ``llvm-min-tblgen``. It is implied above.